lib/vsc/filesystem/posix.py

```diff
@@ -350,8 +350,7 @@ def make_dir(self, obj=None):
         obj = self._sanity_check(obj)
         try:
             if self.dry_run:
-                self.log.info("Making directory %s." % (obj),
-                    "Dry-run, so not really doing anything. Pretending it did succeed though. Returning True.")
+                self.log.info("Dry-run: pretending to create directory %s" % obj)
```

Comment:
use the logging templating `self.log.info("Dry-run: pretending to create directory %s", obj)`